ciao-launcher/README.md

```diff
@@ -473,12 +473,10 @@ For a full example see
 
 # Attaching and Detaching RBD images
```

Comment:
I guess this title should just be Attaching and not Attaching and Detaching.  Some point soon I'm going to clean up this file, so feel free to ignore this comment.